apps/extension/src/routes/popup/login.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from 'ui/components';
+import { FadeTransition, LoginForm } from '../../components';
+import { PagePath } from '../page/paths';
+
+export const Login = () => {
+  return (
+    <FadeTransition>
+      <div className='grid gap-4 p-6 text-white shadow-sm'>
+        <p className='text-center text-2xl font-semibold leading-none tracking-tight'>
+          Welcome back!
+        </p>
+        <p className='text-center text-sm text-muted-foreground'>A decentralized network awaits</p>
+        <div className='grid gap-4 p-6 pt-0'>
+          <LoginForm />
+          <Button
+            className='text-white'
```

Comment:
This and the support text beneath should be a gray color like `text-muted-foreground`